packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardRenderer.js

```diff
@@ -50,6 +50,22 @@ function isPlainObject(obj) {
   return Object.getPrototypeOf(obj) === Object.prototype;
 }
 
+/**
+ *
+ * @param {* The current active element} element
+ * @param {* The name of attribute to be added to the element properties} qualifiedName
+ * @param {* the target value to set for the attribute} nextValue
+ */
+function setAttribute(element, qualifiedName, nextValue) {
```

Comment:
This seems copying code from the function `setAttributeWithUndo`, can we refactor them?

Reply:
removed and will use the original setAttributewithundo